### ISSUE TITLE
Screen resolution inside the VM not resized correctly after double-clicking (to Zoom) the title bar of the VM window

### ISSUE THREAD

**Issue by rxhfcy** [utmapp/UTM#5703]
**Describe the issue**
Screen resolution inside the VM is not resized correctly after double-clicking (to Zoom) the title bar of the VM window

(when the host macOS has the following setting: in System Settings... / Desktop & Dock / "Double-click a window's title bar is" -> "Zoom")

**Steps to reproduce**
1. Make a new Fedora Linux VM (New VM, virtualize, aarch64, newest is currently [Fedore 38 aarch64](https://download.fedoraproject.org/pub/fedora/linux/releases/38/Workstation/aarch64/iso/Fedora-Workstation-Live-aarch64-38-1.6.iso))
2. After installing (default settings), inside the VM: Settings / Displays
3. Resize the VM window (make it as small as possible by dragging the VM window borders)
4. Look at "Resolution" inside the VM: 800 x 600 or something like that
5. Resize the VM window (make it a little bit taller) by dragging the VM window border
6. Look at "Resolution": now it's 800 x 627 or whatever
7. (in host macOS, make sure that System Settings... / Desktop & Dock / "Double-click a window's title bar to" -> "Zoom")
8. Double-click the host VM window title bar to "Zoom" (to make the window bigger)

What happens: the VM now has black borders inside its window. Guest VM resolution didn't change.
What I expected: behave exactly like if I had manually made the window bigger by resizing it by dragging the VM window's borders (no black borders inside VM window, change the guest resolution normally)

Screen shot of what happened after step 8. (black borders, resolution didn't change)
<img width="1392" alt="black_borders" src="https://github.com/utmapp/UTM/assets/42669366/5f429ba1-8699-453c-9371-dfd9e345cdeb">

**Configuration**  
* UTM Version: Version 4.3.5 (87)
* macOS Version: 14.0 Beta (23A5337a)
* Mac Chip (Intel, M1, ...): M1

**Comment by osy** [comment]
Can you try v4.4 beta, I think this is related to #5577 which was fixed.

**Comment by osy** [comment]
I think it would be even better if you can provide more details and maybe in an new issue? There's no way it'll be fixed in the new beta if I don't know what the issue is.

**Comment by osy** [comment]
> the VM doesn't work at all on the v4.4 beta

What does this mean? You pressed start and nothing happened? Did it crash and you have a crash log? Did you see an error message?

**Comment by osy** [comment]
Okay I understand the part about "didn't start at all" but I need to know *precisely* what that means. Ideally provide a screen recording if you don't know how to explain it. Did you see any error message? Did you see a crash report dialog pop up?

**Comment by osy** [comment]
Yes, that would be useful. Attach the debug.log and the screenshot from above please.

**Comment by osy** [comment]
No worries, I will file it then
